toonz/sources/translations/german/toonzlib.ts

```diff
@@ -0,0 +1,610 @@
+        <translation>FX gruppieren</translation>
+    </message>
+    <message>
+        <source>Ungroup Fx</source>
+        <translation>FX-Gruppe aufheben</translation>
+    </message>
+    <message>
+        <source>Rename Group  : %1 &gt; %2</source>
+        <translation>Gruppe umbenennen : %1 &gt; %2</translation>
+    </message>
+    <message>
+        <source>Set Keyframe</source>
+        <translation>Keyframe setzen</translation>
+    </message>
+    <message>
+        <source>Remove Keyframe</source>
+        <translation>Keyframe löschen</translation>
+    </message>
+    <message>
+        <source>Cycle</source>
+        <translation>Durchschalten</translation>
+    </message>
+    <message>
+        <source>The autocentering failed on the current drawing.</source>
+        <translation>Das automatische Zentrieren ist bei der aktuellen Zeichnung fehlgeschlagen</translation>
+    </message>
+    <message>
+        <source>Move</source>
+        <translation>Verschieben</translation>
+    </message>
+    <message>
+        <source>Edit Rotation</source>
+        <translation>Rotation bearbeiten</translation>
+    </message>
+    <message>
+        <source>Move E/W</source>
+        <translation>X verschieben</translation>
+    </message>
+    <message>
+        <source>Move N/S</source>
+        <translation>Y verschieben</translation>
+    </message>
+    <message>
+        <source>Move Z</source>
+        <translation>Z verschieben</translation>
+    </message>
+    <message>
+        <source>Edit Stack Order</source>
+        <translation>Stack-Reihenfolge bearbeiten</translation>
+    </message>
+    <message>
+        <source>Edit Scale W</source>
+        <translation>Breite skalieren</translation>
+    </message>
+    <message>
+        <source>Edit Scale H</source>
+        <translation>Höhe skalieren</translation>
+    </message>
+    <message>
+        <source>Edit Scale</source>
+        <translation>Größe bearbeiten</translation>
+    </message>
+    <message>
+        <source>Edit PosPath</source>
+        <translation>Pos-Pfad bearbeiten</translation>
+    </message>
+    <message>
+        <source>Edit Shear X</source>
+        <translation>X-Scherung bearbeiten</translation>
+    </message>
+    <message>
+        <source>Edit Shear Y</source>
+        <translation>Y-Scherung bearbeiten</translation>
+    </message>
+    <message>
+        <source>%1  %2  Frame : %3</source>
+        <translation>%1  %2  Frame : %3</translation>
+    </message>
+    <message>
+        <source>Set Keyframe   %1 at frame %2</source>
+        <translation>Keyframe %1 bei Frame %2 setzen</translation>
+    </message>
+    <message>
+        <source>Remove Keyframe   %1 at frame %2</source>
+        <translation>Keyframe %1 bei Frame %2 entfernen</translation>
+    </message>
+    <message>
+        <source>Move Center   %1  Frame %2</source>
+        <translation>Verschiebe Mittlere-Position %1 Frame %2</translation>
```

Comment:
I've changed it to "Zentrum" for now, since neither translation makes any reference to an anchor point. Checking the program should clear up the exact function